test/TestUtilities/Test.Utility/Protocol/TestHttpSource.cs

```diff
@@ -62,14 +62,14 @@ public override Task<T> GetAsync<T>(HttpSourceCachedRequest request, Func<HttpSo
             return base.GetAsync(request, processAsync, log, token);
         }
 
-        protected override Stream TryReadCacheFile(string uri, TimeSpan maxAge, string cacheFile)
+        protected override (Stream, bool isExpired) TryReadCacheFile(TimeSpan maxAge, string cacheFile)
         {
             if (DisableCaching)
             {
-                return null;
+                return (null, false);
```

Comment:
Here explicitly disabling cache, in this case returning `false` for expired property, but this value doesn't matter latter because we have another `bypass` property detect this.